packages/client/package.json

```diff
@@ -36,6 +36,7 @@
     "@types/make-fetch-happen": "^10.0.0"
   },
   "dependencies": {
+    "@sigstore/bundle": "^0.0.0",
```

Comment:
Do we not want to publish an actual version of this package? Or follow up once we publish sigstore-js?

Reply:
Yeah, I'll be bumping to 1.0 before I cut a release with any of this stuff